### PR TITLE
Moving VTK output to `BcubeVTK`

### PR DESCRIPTION
**Pull request by bmxam**
VTK output has been moved to the dedicated project `BcubeVTK`. `cell_mean` is now exported and works for any `AbstractLazy`.

Between the three PRs, this is the first that shall be merged (because it introduces version 0.2.0).

I suggest to increase the "minor" version number to "2" (i.e 0.2.0) because this is an important breaking change. I also think that we shall, from now on, respect more this convention of increasing the number when breaking changes occur.

### CODE REVIEW

**Comment by bmxam** [comment]
Will close #70

**Comment by bmxam** [comment]
@ghislainb how can the "IntegrationTest" be successfull since `write_vtk_*` doesn't exist anymore?

**Comment by ghislainb** [comment]
> @ghislainb how can the "IntegrationTest" be successfull since `write_vtk_*` doesn't exist anymore?

may be because of [compat](https://github.com/bcube-project/BcubeTutorials.jl/blob/e5ef3e24c578e5cc3fb277cf08cfac89d057055d/Project.toml#L23C1-L24C17)

File: src/feoperator/projection_newapi.jl [hunk 1]
```diff
@@ -73,10 +73,12 @@ get_fespace(cache::CellMeanCache) = cache.feSpace
 get_measure(cache::CellMeanCache) = cache.measure
 get_mass_matrix(cache::CellMeanCache) = cache.massMatrix
 
-function build_cell_mean_cache(u::AbstractSingleFieldFEFunction, dΩ::Measure)
+function build_cell_mean_cache(u::AbstractLazy, dΩ::Measure)
     mesh = get_mesh(get_domain(dΩ))
     fs = FunctionSpace(:Lagrange, 0)
-    ncomp = get_ncomponents(get_fespace(u))
+    _, N = get_return_type_and_codim(u, mesh)
+    @assert length(N) == 1 "ncomponents > 1 not supported yet"
```
Comment (ghislainb) [inline src/feoperator/projection_newapi.jl:80]: Does [it](https://github.com/bcube-project/BcubeTutorials.jl/blob/e5ef3e24c578e5cc3fb277cf08cfac89d057055d/src/example/euler_naca_steady/euler_naca_steady.jl#L239) still work with this limitation?

Comment (bmxam) [reply src/feoperator/projection_newapi.jl:80]: It does. It's working for vectors (only one "axis") but will fail for tensors.

**Review by ghislainb — CHANGES_REQUESTED** [review]

File: src/io/io_interface.jl [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
-    read_file([handler::AbstractIoHandler,] filepath::String; domainNames = nothing, varnames = nothing, topodim = 0, kwargs...,)
+    read_file(
+        [handler::AbstractIoHandler,]
+        filepath::String;
+        domainNames = String[],
+        varnames = nothing,
+        topodim = 0,
+        spacedim = 0,
+        kwargs...,
+    )
```
[the remark this answers is not in the document]
Comment (bmxam) [reply src/io/io_interface.jl:15]: done

File: src/io/io_interface.jl [hunk 3]
```diff
@@ -86,14 +92,39 @@ Write a set of `AbstractLazy` to a file.
...
+```
 
+# Remarks:
+* If `mesh` is of degree `d`, the solution will be written on a mesh of degree `mesh_degree`, even if this
+number is different from `d`.
+* The degree of the input FEFunction (P1, P2, P3, ...) is not used to define the nodes where the solution is
+written, only `mesh` and `mesh_degree` matter. The FEFunction is simply evaluated on the aforementionned nodes.
 
-# Implementation
+# Dev notes
```
[the remark this answers is not in the document]
Comment (bmxam) [reply src/io/io_interface.jl:123]: done

**Review by ghislainb — APPROVED** [review]

**Pull request merged**